mock/isbehandlerdialog/mockIsbehandlerdialog.ts

```diff
@@ -19,6 +23,21 @@ export const mockIsbehandlerdialog = (server: any) => {
       }
     }
   );
+  server.get(
+    `${ISBEHANDLERDIALOG_ROOT}/melding/:uuid/:vedleggNumber/pdf`,
+    Auth.ensureAuthenticated(),
+    (req: express.Request, res: express.Response) => {
+      res.type("application/pdf");
```

Comment:
Man måtte sette dette eksplisitt i mockendepunktet for å få riktig contentType. Fungerer derimot ikke lokalt likevel :/

Reply:
Tror det skal fungere lokalt nå - ref kommentar over.